packages/sdk/utils/src/account.ts

```diff
@@ -131,6 +131,18 @@ export function generateKeysFromSeed(
   }
 }
 
+function isLatinBasedLanguage(language: MnemonicLanguages) {
+  if (
+    language === MnemonicLanguages.chinese_simplified ||
+    language === MnemonicLanguages.chinese_traditional ||
+    language === MnemonicLanguages.japanese ||
+    language === MnemonicLanguages.korean
+  ) {
```

Comment:
I think (not really sure about this), that as the english is a germanic one, it has not accents, so you could add it too

Reply:
I think that's true but I didn't include it just in case a user with an English mnemonic accidentally enters a letter with an accent